core/src/test/scala/neotypes/AlgorithmSpec.scala

```diff
@@ -51,7 +49,7 @@ final class AlgorithmSpec[F[_]](testkit: EffectTestkit[F]) extends AsyncDriverPr
                 }
               )
            """.query[Unit].execute(d)
-      result <- """CALL gds.triangleCount.mutate('myGraph', {
+        result <- """CALL gds.triangleCount.mutate('myGraph', {
                      mutateProperty: 'tc'
```

Comment:
@BalmungSan the formatting gets tough.  I'm not entirely sure how the extra whitespace is getting added.  Do you think the best way to make the formatting uniform across the project would be to add a scalafmt rule?  This would also ease the formatting burden as new contributors come on to the project and make these same mistakes.

Reply:
@geoffjohn11 would you like to try to add such formatting rules? I guess that would be great most popular projects do that.

Reply:
I messed around this weekend with a few different format files.  None of them really came close to adhering to the format that already exists across the project.

Reply:
@geoffjohn11 yeah I know, but I guess it doesn't matter as long as it is a standard format.
I guess the best would be to open a new PR adding that so we can review how the codebase would change and after we merge that one update this one.